argocd_instance/setting-up-argocd-instance.adoc

```diff
@@ -24,6 +24,7 @@ include::modules/gitops-argo-cd-installation.adoc[leveloffset=+1]
 * link:https://argocd-operator.readthedocs.io/en/latest/usage/routes/#setting-tls-modes-for-routes[Configuring the route TLS termination]
 * xref:../argocd_instance/argo-cd-cr-component-properties.adoc#argo-cd-properties_argo-cd-cr-component-properties[Argo CD custom resource properties]
 * link:https://docs.openshift.com/container-platform/latest/rest_api/network_apis/route-route-openshift-io-v1.html#spec-tls[Specification for TLS termination configuration]
+* link:https://argocd-operator.readthedocs.io/en/latest/usage/ha/repo-server[Persistent volumes for Repo server storage]
```

Comment:
Looking at the target doc, I see "Repo Server" and "repo server", but not "Repo server". Suggest switching it out here for one of those others.

Reply:
Agreed, thanks for the catch. Fixed.